ampbench_lib.js

```diff
@@ -775,10 +775,14 @@ function lib_fetch_cdn_validator_spec_file_revision(callback) {
  */
 function lib_load_validator(opt_force_reload) {
     let _force_reload = opt_force_reload || false;
-    const _validator = amphtml_validator.getInstanceFromFileSync(VALIDATOR_JS_FILE, _force_reload);
-    amphtml_validator_instance = _validator; // cache the module level instance here
-    amphtml_validator_spec_file_revision = lib_extract_validator_spec_file_revision(_validator.scriptContents);
```

Comment:
So far it has been a useful indicator of when to update the cached validator.js via the `../command_force_validator_update` route.

I am open to any other convenient way of tracking validator updates somehow. Perhaps via hashing the validator content? However, the spec file revision has been useful for support/validator skew tracking.